### ISSUE TITLE
Upgrade dialog lacks option to upgrade later

### ISSUE THREAD

**Issue by sethwklein** [keybase/client#6203]
Keybase GUI Version: 1.0.18-20170223162556+af0e640

<img width="623" alt="screen shot 2017-03-09 at 6 31 26 pm" src="https://cloud.githubusercontent.com/assets/3307616/23775436/ab428300-04f6-11e7-8dc3-4e73fdc06270.png">

There's three things I might want to do here:
1. Upgrade now
2. Never upgrade
3. Upgrade when I have more bandwidth

This dialog has buttons for the first two, but not the third. Minimize is a common choice for the third. Not making the dialog be always on top also works.

**Comment by otteydw** [comment]
I'd also suggest that the default should not be "Update."   The problem is when you happen to be typing away and not notice that a popup has come on your screen.  Then you hit space or enter, and "Update" has been selected.  If "remind me later" was an available default option, no-harm, no-foul.

**Comment by cjb** [comment]
@otteydw Hm, why is Update a bad default when you're typing away, though? What's the harm?

**Comment by otteydw** [comment]
@cjb   What if you didn't want to update?  Maybe like @sethwklein suggested, you want to wait until you're on a higher bandwidth connection.  Or maybe you want to avoid the update because you read somewhere that the latest version had a known issue.  I'm not a UX designer by any stretch, but I always feel like "make a change" should not be the default.

**Comment by cjb** [comment]
Thanks.

I'm not a designer either, but I'd go with something more like "if one action is the right one 99% of the time and doesn't come with any real risk, that one should be default".

I hadn't thought about wanting to wait for more bandwidth, I agree that's a "risk" of a sort.

**Comment by sethwklein** [comment]
Yeah, if I'm on a metered connection (tethered), the upgrade button might as well read "waste real dollars." For a 100MB upgrade on cell data, it's typically 1 USD, and I may not be able to buy an unlimited amount.

Generally, it's bad UX to steal focus with something unrelated to whatever the user is doing, but it seems to be how all these application specific updaters work. I suppose you could try to do better than the herd and light the tray icon (or whatever they call it on OS X.)

**Comment by sethwklein** [comment]
I'm missing yesterday's update. The window popped up at a time when I couldn't leave it on top and couldn't upgrade so I had to click skip version :( There's still no upgrade later option, and no way to trigger an upgrade check manually. :(
